src/ipinfo.rs

```diff
@@ -41,6 +47,8 @@ impl Default for IpInfoConfig {
             token: None,
             timeout: Duration::from_secs(3),
             cache_size: 100,
+            country_file_path: None,
+            eu_file_path: None,
```

Comment:
Shouldn't we provide the `./src/eu.json` string right here?

Reply:
I tried following the existing convention as used in the token, if the user wants to add a custom path then they can add it while creating a new client otherwise they can leave everything to `None` and we are setting the path if we receive `None`. I think this approach is easier for users as it's better to leave the path as `None` rather than setting the path for every new client. What do you say? if the other way is more easier than we can go with that approach.

Reply:
The thing is like 99% of use cases won't override this variable if we set it to the existing `./src/*.json` paths, that's why I'm saying it'd be better to initialize it as such right here.